src/EditorFeatures/Core.Wpf/BackgroundWorkIndicator/WpfBackgroundWorkIndicatorFactory.cs

```diff
@@ -21,21 +21,18 @@ namespace Microsoft.CodeAnalysis.Editor.BackgroundWorkIndicator
     internal sealed partial class WpfBackgroundWorkIndicatorFactory : IBackgroundWorkIndicatorFactory
```

Comment:
The platform implementation should be cross platform. Could you make this the default implementation now and delete your other context?